src/api/app/views/webui/groups/_group_members.html.haml

```diff
@@ -9,9 +9,16 @@
       - group.users.each do |user|
         .card.m-1.p-2.group-user
           - if write_access
-            = render(partial: 'webui/groups/group_members_delete_dialog', locals: { user: user, group: group })
-            = link_to('#', data: { toggle: 'modal', target: "#delete-group-members-modal-#{dom_id(user)}" },
-                      title: 'Remove member from group') do
+            = render DeleteConfirmationDialogComponent.new(modal_id: 'delete-group-members-modal',
+                                                           method: :delete,
+                                                           options: { modal_title: 'Do you want to remove this group member?' })
```

Comment:
Rendering the component could happen only once if it is moved outside the `group.users.each do |user|` loop.

Reply:
you are right, I'll change it

Reply:
Now I render the component after the loop.